src/OrchardCore.Modules/OrchardCore.Deployment/Views/Items/ContentDeploymentStep.Fields.Summary.cshtml

```diff
@@ -1,8 +1,8 @@
 @model dynamic
```

Comment:
Remove dynamic, use the correct view model

Reply:
This should be `ShapeViewModel<ContentDeploymentStep>`, then you can remove the cast too.